src/main/java/rockPaperScissors/player/Computer.java

```diff
@@ -0,0 +1,31 @@
+package rockPaperScissors.player;
+
+import rockPaperScissors.move.Move;
+import rockPaperScissors.move.MoveFactory;
+import rockPaperScissors.move.Moves;
+import rockPaperScissors.ui.Ui;
+
+import java.util.List;
+import java.util.Random;
+
+public class Computer implements Player {
+
+    private final MoveFactory moveFactory;
+    private String name;
+
+    public Computer() {
+        this.name = "computer";
+        moveFactory = new MoveFactory();
+    }
+
+    public Move makeMove(Ui ui) {
+        List<String> possibleMovesMoves = Moves.possibleMoves();
+        String randomMove = possibleMovesMoves.get(new Random().nextInt(possibleMovesMoves.size()));
```

Comment:
You could move the entire randomness thing int Moves itself, so here you could then write `Moves.randomMove()` 😄